src/components/SearchNewTabSwitch/SearchNewTabSwitch.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { createStyles, Switch, Group } from '@mantine/core';
+import { useTranslation } from 'next-i18next';
+import { useConfig } from '../../tools/state';
+
+const useStyles = createStyles((theme) => ({
+  root: {
+    position: 'relative',
+    '& *': {
+      cursor: 'pointer',
+    },
+  },
+
+  icon: {
+    pointerEvents: 'none',
+    position: 'absolute',
+    zIndex: 1,
+    top: 3,
+  },
+
+  iconLight: {
+    left: 4,
+    color: theme.white,
+  },
+
+  iconDark: {
+    right: 4,
+    color: theme.colors.gray[6],
+  },
+}));
+
+export function SearchNewTabSwitch() {
+  const { config, setConfig } = useConfig();
+  const { classes, cx } = useStyles();
+  const defaultPosition = config?.settings?.searchNewTab ?? true;
+  config.settings.searchNewTab = defaultPosition;
+  const [widgetPosition, setWidgetPosition] = useState<boolean>(defaultPosition);
```

Comment:
``widgetPosition``? Did you copy this name? You should change it to ``openInNewTab`` or something similar

Reply:
Yes, the name is copied, but I thought widget was it was referring to the switch itself. I'll change it.